packages/cli/test/commands/fix/__snapshots__/fix.test.ts.snap

```diff
@@ -42,15 +43,13 @@ These rules, which compare the behavior of the automaton to real life, can be co
   All other live cells die in the next generation. Similarly, all other dead cells stay dead.
 */
 
-// @ts-expect-error @rehearsal TODO TS2307: Cannot find module 'node:process' or its corresponding type declarations.
```

Comment:
This is fixed by https://github.com/rehearsal-js/rehearsal-js/pull/1102/files#diff-9db2b41974dec78bf1531a30673ff621b115da09c4c5c943e606204bca78e381R106